content/companies.md

```diff
@@ -6,27 +6,29 @@ title: Information to companies
 The directory is run by the non-profit organization __2factorauth__, registered in Sweden.
 
 ## The content
-All our content is crowdsourced and moderated by __2factorauth__. If a person requests to list a new service and said service meets our [listing criteria][criteria] and [guidelines][guidelines], it gets published to the directory.
+All our content is crowdsourced and moderated by __2factorauth__. If a person requests to list a new service that meets our [listing criteria][criteria] and [guidelines][guidelines], it gets published in the directory.
 
-If you find your service listed here it most likely means one of your own users have requested it. If you already provide two factor authentication, the list will function as free promotion for your service.  
-If you currently do not support two factor authentication, the listing functions as a reminder that your customers think this is an important thing for you to provide. 
+If you find your service listed here, one of your users has requested it. If you already provide two-factor authentication, the list will function as a free promotion for your service.    
+If you currently do not support two-factor authentication, the listing serves as a reminder that your customers think this is important for you to provide.
 
 ### Edit a listing
-If the data about a service is inaccurate you can create an [issue][issue] or [pull request][pr] on GitHub to update it.
+If the data about a service is inaccurate, you can create an [issue][issue] or [pull request][pr] on GitHub to update it.
 
-## Takedown requests / DMCA
-As __2factorauth__ is registered in Sweden, The Digital Millennium Copyright Act does not apply to 2fa.directory and thus DMCA requests will be ignored.
-All the information on 2fa.directory is both public information and compliant with _fair use_. We believe consumers should be able to compare services to find the alternative that best fits them; We therefore do not remove listings.
+## Takedown requests / DMCA[api.md](api.md)
```

Comment:
Why is the API page linked here?

Reply:
Looks like I accidentally created a link instead of opening the file in the IDE :facepalm: